picard/util/tags.py

```diff
@@ -32,56 +32,79 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 
+from collections import OrderedDict
 from collections.abc import MutableSequence
 import re
 
+
+try:
+    from markdown import markdown
+except ImportError:
+    markdown = None
+
 from picard.i18n import (
     N_,
     gettext as _,
 )
 
 
+TEXT_NOTES = N_('Notes:')
+TEXT_NO_DESCRIPTION = N_('No description available.')
+
+ATTRIB2NOTE = OrderedDict(
+    is_preserved=N_('preserved read-only'),
+    not_script_variable=N_('not for use in scripts'),
+    is_calculated=N_('calculated'),
+    is_file_info=N_('info from audio file'),
+    not_from_mb=N_('not provided from MusicBrainz data'),
+)
+
+
 class TagVar:
     def __init__(
         self, name, shortdesc=None, longdesc=None,
-        is_preserved=False, is_hidden=False, is_script_variable=True,
-        is_tag=True, is_calculated=False, is_file_info=False
+        is_preserved=False, is_hidden=False, not_script_variable=False,
+        is_tag=True, is_calculated=False, is_file_info=False,
+        not_from_mb=False
     ):
         """
         shortdesc: Short description (typically one or two words) in title case that is suitable
                    for a column header.
         longdesc: Brief description in sentence case describing the tag/variable.  This should
                   be similar (within reasonable length constraints) to the description in the Picard User
-                  Guide documentation, and could be used as a tooltip when reviewing a script.
+                  Guide documentation, and could be used as a tooltip when reviewing a script.  May
+                  contain markup.
         is_preserved: the tag is preserved (boolean, default: False)
         is_hidden: the tag is "hidden", name will be prefixed with "~" (boolean, default: False)
-        is_script_variable: the tag can be used as script variable (boolean, default: True)
+        not_script_variable: the tag cannot be used as script variable (boolean, default: False)
```

Comment:
Actually I think we should keep `is_*` for parameters, it would be more consistent, but we can add derived properties for `ATTRIB2NOTE` purpose:

```python
class TagVar:
def __init__(
        self, name, shortdesc=None, longdesc=None,
        is_preserved=False, is_hidden=False, is_script_variable=True,
        is_tag=True, is_calculated=False, is_file_info=False,
        is_from_mb=True,
    ):
    ....

    @property
    def not_from_mb(self):
        return not self.is_from_mb

     @property
     def not_script_variable(self):
        return not self.is_script_variable

```